web/package.json

```diff
@@ -28,7 +28,10 @@
     "eslint:fix": "eslint . --fix",
     "eslint": "eslint .",
     "stylelint": "stylelint --config stylelint.config.js \"src/**/*.{ts,tsx,css}\"",
-    "ts:check": "tsc --build"
+    "ts:check": "tsc --build",
+    "prettier:check": "yarn prettier --ignore-path  ../.prettierignore --check .",
+    "prettier:fix": "yarn prettier --ignore-path  ../.prettierignore --write .",
+    "circleci:update-config": "yarn --cwd .. circleci:update-config"
```

Comment:
What's this doing here? And what does --cwd do?

Reply:
Setting the working directory from which to execute the command. I thought it was nice that you can just update the circleci config without having to switch directories manually.